src/include/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ install(FILES
     zlog/stream.h
     zlog/options.h
     zlog/statistics.h
+    zlog/mempool/mempool.h
```

Comment:
Do these headers need to be installed?

Reply:
They may need to be installed if the have things that installed headers depend on. If they are strictly used internally, they don't need to be installed.